openstack/compute/v2/servers/fixtures.go

```diff
@@ -1,7 +1,7 @@
-// +build fixtures
```

Comment:
Alternatively, to run all the Gophercloud unit tests, there is a script in the root directory.
Run `script/unittest` from the command-line